### ISSUE TITLE
Mentions does't triggered on android chrome 73^

### ISSUE THREAD

**Issue by sadwizard** [signavio/react-mentions#320]
On mobile chrome after update version on 73.0.3683.90 not fired mentions list.
For example i defined mention with trigger @ then on mobile trying to typing @ and nothing is happen

**Comment by marinona** [comment]
I've tested in several mobile devices and I've detected different behaviors. 
As @sadwizard points out, in some devices mentions triggering doesn't work at all. 
In other devices, the dropdown is only triggered after a space (so when writting the second word).
In some cases writting "@" triggers the dropdown with all options but writting after that doesn't filter the results.

**Comment by nikita2423** [comment]
Yes same results on my side,  when we write any character after @ and then press backspace then it is getting triggered.
Any updates ?

**Comment by marinona** [comment]
> Yes same results on my side, when we write any character after @ and then press backspace then it is getting triggered.
> Any updates ?
Yes, it's solved in the last version! 
Solution merged: https://github.com/signavio/react-mentions/pull/402

**Comment by nikita2423** [comment]
> > Yes same results on my side, when we write any character after @ and then press backspace then it is getting triggered.
> > Any updates ?
> 
> Yes, it's solved in the last version!
> Solution merged: #402

Thanks!
